dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/GleifApiAccessor.kt

```diff
@@ -0,0 +1,58 @@
+package org.dataland.datalandbatchmanager.service
+
+import org.apache.commons.io.FileUtils
+import org.dataland.datalandbatchmanager.service.CompanyUpload.Companion.MAX_RETRIES
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.io.File
+import java.io.FileNotFoundException
+import java.net.SocketException
+import java.net.URL
+
+/**
+ * The class to download the zipped GLEIF golden copy CSV files
+ */
+@Component
+class GleifApiAccessor {
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private val gleifBaseUrl = "https://goldencopy.gleif.org/api/v2/golden-copies/publishes/lei2"
```

Comment:
Passt besser in die application properties finde ich. Was meinst du?

Reply:
Gleich gepusht